saleor/checkout/base_calculations.py

```diff
@@ -237,12 +238,49 @@ def base_checkout_subtotal(
     return sum(line_totals, zero_money(currency))
 
 
+def checkout_total(
+    checkout_info: "CheckoutInfo",
+    discounts: Iterable[DiscountInfo],
+    lines: Iterable["CheckoutLineInfo"],
+) -> Money:
+    """Return the total cost of the checkout including discounts and vouchers."""
```

Comment:
Is this a function that should be called, when we don't have any active flat rate/tax plugin/tax app? If yes, maybe it would be worth mentioning this in the description. This could be hard to understand what kind of function should be used in a specific scenario.